src/hermes/model/context.py

```diff
@@ -173,6 +173,9 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         if exc_type is not None and issubclass(exc_type, HermesValidationError):
             exc = traceback.TracebackException(exc_type, exc_val, exc_tb)
             self._base.error(self._ep, exc)
+            self._log.warning("{}: {}".format(
+                exc_type.__name__,
+                ' '.join(str(x) for x in exc_val.args)))
```

Comment:
```suggestion
            self._log.warning("%s: %s",
                exc_type,
                ' '.join(map(str, exc_val.args)))
```

While your change is totally fine, this proposal makes it more "pythonic" and slightly improves performance:

- Using `self._log.warning("%s: %s", ...)` leads to evaluation of the format string only in cases where the log message is actually rendered (i.e., if it is not filtered out by the logger or handler)
- Similar using `map(str, ...)` is only evaluated once the content is actually iterated. However, this is in fact already done when `' '.join(...)` is called and thus does not improve performance a lot here...